geom.js

```diff
@@ -1,26 +1,75 @@
 class Rectangle {
-  constructor(length, width) {
-    this.length = length;
-    this.width = width;
+  constructor (length, width) {
+    this.length = length
+    this.width = width
+  }
+  isSquare () {
+    return this.length === this.width
+  }
+  area() {
+    return this.length * this.width
+  }
+  perimeter() {
+    return (this.length * 2) + (this.width * 2)
   }
 }
 
-
 class Triangle {
-  constructor(sideA, sideB, sideC){
-    this.sideA = sideA;
-    this.sideB = sideB;
-    this.sideC = sideC;
+  constructor (sideA, sideB, sideC) {
+    this.sideA = sideA
+    this.sideB = sideB
+    this.sideC = sideC
+  }
+  isEquilateral() {
+    if (this.sideA === this.sideB && this.sideB === this.sideC){
```

Comment:
Since the statement `this.sideA === this.sideB && this.sideB === this.sideC` evaluates to a `true`/`false` value, we can directly return that result instead of wrapping it in an `if` statement an explicitly return `true` or `false`:

```js
return  this.sideA === this.sideB && this.sideB === this.sideC
```